fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -603,6 +603,9 @@ private void deliverExecFragments() throws Exception {
 
     private void deliverExecFragmentRequests(boolean enablePipelineEngine) throws Exception {
         long queryDeliveryTimeoutMs = Math.min(queryOptions.query_timeout, queryOptions.query_delivery_timeout) * 1000L;
+        if (queryOptions.query_delivery_timeout == 0) {
+            queryDeliveryTimeoutMs = queryOptions.query_timeout * 1000L;
```

Comment:
This may too short for loading jobs and is Incompatible with the original implementation:
In our previous implementation based on jprotobuf, queryDeliveryTimeoutMs equals 0 means the below call to `Future.get()` will never timed out, although I think it's bug of jprotobuf, better keep this behavior now.